tests/functional/metadataProxy/routesToMem.js

```diff
@@ -34,13 +34,13 @@ const bucketInfo = {
     deleted: false,
     serverSideEncryption: null,
     versioningConfiguration: null,
-    websiteConfiguration: null,
```

Comment:
💭I don't follow why we need to remove this.

Reply:
It appears to not be served back to the client